lib/detail_page.dart

```diff
@@ -0,0 +1,54 @@
+import 'package:flutter/material.dart';
+
+import 'api/book.dart';
+
+/// Detail Page that displays information on the book
+class DetailPage extends StatelessWidget {
+  /// Creates a Detail Page and ensures book is not null
+  const DetailPage({@required this.book})
+      : assert(book != null, 'book cannot be null');
+
+  /// The book being displayed
+  final Book book;
+
+  @override
+  Widget build(BuildContext context) => Scaffold(
+      appBar: AppBar(title: Text(book.title)),
+      body: ListView(
+        children: <Widget>[
+          Image.network(book.bookImage),
+          Container(
+            padding: const EdgeInsets.only(left: 32, right: 32),
+            child: Row(children: <Widget>[
+              Expanded(
+                child: Column(
+                  crossAxisAlignment: CrossAxisAlignment.start,
+                  children: <Widget>[
+                    Container(
+                      padding: const EdgeInsets.only(bottom: 8),
+                      child: Text(
+                        book.title,
+                        style: TextStyle(fontWeight: FontWeight.bold),
+                      ),
+                    ),
+                    Text(book.author, style: TextStyle(color: Colors.grey[500]))
+                  ],
+                ),
+              ),
+              Text('\$ ${book.price}')
+            ]),
+          ),
+          Container(
+              padding: const EdgeInsets.only(right: 32, left: 32, top: 15),
+              child: Text(book.description)),
+          Container(
+              padding: const EdgeInsets.only(right: 32, left: 32, top: 10),
+              child: ButtonTheme(
+                  minWidth: 10,
```

Comment:
Instead of hard coding the `minWidth` you can do `width: double.infinity`. This achieves the same thing without using hard coded values.